examples/python/fetch_l2_acc_details.py

```diff
@@ -0,0 +1,40 @@
+import asyncio
+import logging
+import os
+from typing import Dict, List
+
+from utils import (
+    generate_paradex_account,
+    get_l1_eth_account,
+    get_paradex_config,
+)
+
+paradex_http_url = "https://api.testnet.paradex.trade/v1"
```

Comment:
Preferably re-use URL defined in `utils.py`

Reply:
done!! this was not being used - so removed!